data/types.go

```diff
@@ -115,6 +115,7 @@ type Root struct {
 	Custom      *json.RawMessage      `json:"custom,omitempty"`
 
 	ConsistentSnapshot bool `json:"consistent_snapshot"`
+	NewRoot            bool `json:"-"`
```

Comment:
Could you add a comment for why we have this variable, and why we do not serialise it to JSON?

Reply:
Should I add another comment for `NewTargets` too?